packages/mobile/e2e/src/usecases/Send.js

```diff
@@ -18,8 +26,10 @@ export default Send = () => {
     await inputNumberKeypad(AMOUNT_TO_SEND)
     await element(by.id('Review')).tap()
 
+    await sleep(1000)
```

Comment:
I see comments elsewhere around "if we aren't sleeping long enough, this test will fail".

Is there a way to avoid `sleep` in this case and instead wait for some explicit state (with a timeout)?  E.g., wait for a specific element to appear or change state.

Reply:
Locally the tests will pass without the sleep, but in running in Circle Ci the sleeps are needed to reduce test flake. This can be reassessed when setting up the new CICD pipeline.